cli/portforward_test.go

```diff
@@ -160,21 +124,27 @@ func TestPortForward(t *testing.T) {
 			inv.Stdin = pty.Input()
 			inv.Stdout = pty.Output()
 			inv.Stderr = pty.Output()
+
+			iNet := newInProcNet()
+			inv.Net = iNet
```

Comment:
Looks like GitHub lost one of my comments. 😔

I had asked about what `i` in `iNet` means (perhaps `testNet`, `fakeNet`, `memNet`, etc would be more descriptive).

I also suggested a name change => `s/Proc/Mem/` to better match our other implementations.

Since this got merged already, I won't demand you make the changes (just a small wish 😄)